acta-orthopaedica-belgica.csl

```diff
@@ -0,0 +1,161 @@
+    <link href="http://journals.sfu.ca/actaob/index.php/actaob/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0001-6462</issn>
+    <updated>2017-07-06T21:52:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" font-weight="bold" delimiter=",">
+      <name delimiter-precedes-et-al="always" delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator="  "/>
+      <label plural="always" text-case="capitalize-first" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize="true" delimiter=", " delimiter-precedes-last="always" name-as-sort-order="all" sort-separator=" " form="long"/>
+      <label form="long" prefix=", " plural="contextual" suffix="."/>
+    </names>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="numeric" variable="issued" range-delimiter="–"/>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="journal-location">
+    <choose>
+      <if type="article-journal article-magazine" match="any">
+        <text variable="volume"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=", ">
+      <text variable="publisher-place" text-case="title"/>
+      <text variable="publisher"/>
+      <text macro="year-date"/>
+    </group>
+  </macro>
+  <macro name="publisher_thesis">
+    <group delimiter=", ">
+      <text variable="publisher-place"/>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="edition_book">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="numeric"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <citation>
+    <sort>
+      <key variable="citation-number" sort="ascending"/>
+    </sort>
+    <layout font-style="italic" delimiter=" ">
+      <text variable="citation-number" strip-periods="true" quotes="false" vertical-align="baseline" prefix="(" suffix=")"/>
+    </layout>
+  </citation>
+  <bibliography et-al-min="4" et-al-use-first="3" second-field-align="flush" entry-spacing="2">
+    <sort>
+      <key macro="author"/>
+    </sort>
+    <layout suffix=".">
+      <text variable="citation-number" font-weight="bold" suffix="."/>
+      <text macro="author" text-case="capitalize-first" strip-periods="true" suffix=". "/>
```

Comment:
I don't think you want to run capitalize-first and strip-periods on a whole macro, do you?